commands/clock.go

```diff
@@ -3,3 +3,7 @@ package commands
 import "time"
 
 type Clock func() time.Time
+
+func toString(t time.Time) string {
```

Comment:
Should it be `String()` to implement `Stringer`?

Reply:
We can't add a function to a type that is not defined in the same package. I would have to create an alias for `time.Time`...